src/.nuget/dir.props

```diff
@@ -163,6 +179,11 @@
       <Platform>arm64</Platform>
     </OfficialBuildRID>
   </ItemGroup>
+  <ItemGroup Condition="$(SupportedPackageOSGroups.Contains(';Android;'))">
+    <OfficialBuildRID Include="android.21-arm64">
```

Comment:
OfficialBuildRID is only used for platform targets that the official .NET Core build is going to support and we should add it only when we enable building for it. Otherwise, this is not required.

@cydhaselton Can you please undo this change for now?